superset/config.py

```diff
@@ -1436,6 +1436,18 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
     },
 }
 
+# ------------------------------
+# SSH Tunnel
+# ------------------------------
+# Allow users to enable ssh tunneling when creating a DB.
+# Users must check whether the DB engine supports SSH Tunnels
+# otherwise enabling this flag won't have any effect on the DB.
+SSH_TUNNEL_ENABLE = False
```

Comment:
I would make this one a feature flag, and the below one a config, like you have it.

Reply:
So, this one can go in `DEFAULT_FEATURE_FLAGS` and I would just name it the name of the feature. In other words, you can drop "enable" from the name.

Reply:
oh cool! I didn't know about these `DEFAULT_FEATURE_FLAGS`. Thanks!